src/pypgx/Dockerfile

```diff
@@ -0,0 +1,50 @@
+FROM ubuntu:24.04
+
+# Install dependencies and create venv
+RUN apt-get update && \
+    apt-get install --no-install-recommends -y python3.12 python3.12-venv python3.12-dev python3-pip openjdk-17-jdk git build-essential wget unzip && \
+    ln -sf /usr/bin/python3.12 /usr/bin/python3
+
+# Create venv and install pip
+RUN python3 -m venv --without-pip /opt/venv && \
+    python3 -m pip install --target=/opt/venv/lib/python3.12/site-packages pip setuptools wheel && \
+    echo '#!/bin/bash\n/opt/venv/bin/python -m pip "$@"' > /opt/venv/bin/pip && \
+    chmod +x /opt/venv/bin/pip
+
+# Install pypgx
+RUN git clone https://github.com/sbslee/pypgx /opt/pypgx && \
+    cd /opt/pypgx && \
+    git switch 0.26.0-dev && \
```

Comment:
1) Do we really want to run with dev branch?
2) why not to use --branch 0.26.0-dev as below?
3) also here, extract the branch to docker ARG

Reply:
Because there isn't a 0.26.0-dev branch in the bundle. It is a dev branch, so the bundle wasn't updated yet